### PR TITLE
feat(display-area): Add className for stdout, stderr and traceback

### PR DESCRIPTION
**Pull request by lgeiger**
Enables custom styling depending on the output.

Required for https://github.com/nteract/hydrogen/pull/772
Requires https://github.com/nteract/ansi-to-react/pull/25

### CODE REVIEW

File: packages/display-area/src/output.js [hunk 1]
```diff
@@ -44,20 +44,23 @@ export default function Output(props: Props): ?React.Element<any> | null {
     }
     case "stream": {
       const text = output.get("text");
-      switch (output.get("name")) {
+      const name = output.get("name");
+      switch (name) {
         case "stdout":
         case "stderr":
-          return <Ansi>{text}</Ansi>;
+          return <Ansi className={name}>{text}</Ansi>;
```
Comment (rgbkrk) [inline packages/display-area/src/output.js]: This should probably be a fully qualified name like nteract-display-area-stderr

**Review by rgbkrk — APPROVED** [review]

File: packages/display-area/src/output.js (hunk 1 above)
Comment (lgeiger) [reply packages/display-area/src/output.js]: 👍

File: packages/display-area/src/output.js [hunk 2]
```diff
@@ -44,20 +46,29 @@ export default function Output(props: Props): ?React.Element<any> | null {
...
+            className={classPrefix + "traceback"}
+          >{`${output.get("ename")}: ${output.get("evalue")}`}</Ansi>
+        );
       }
-      return <Ansi>{traceback.join("\n")}</Ansi>;
+      return (
+        <Ansi className={classPrefix + "traceback"}>
+          {traceback.join("\n")}
+        </Ansi>
+      );
```
Comment (rgbkrk) [inline packages/display-area/src/output.js:71]: If we ever want coverage here, we should have a test for the traceback. Then again, our jest tests are not connected to our overall coverage. 
¯\\\_(ツ)\_/¯

**Review by rgbkrk — APPROVED** [review]

**Comment by rgbkrk** [comment]
I can go ahead and ship this too for Hydrogen's sake unless you've got some more things.

**Comment by lgeiger** [comment]
> I can go ahead and ship this too for Hydrogen's sake unless you've got some more things.

No rush, I probably won't have time to finish up the Hydrogen branch in the next few days anyway.

**Pull request merged**